website/docs/getting-started/setup/instance-configuration/custom-mongodb-redis.md

```diff
@@ -1,38 +1,83 @@
 ---
 sidebar_position: 3
+toc_max_heading_level: 4
 ---
-# Custom MongoDB & Redis
+# External MongoDB and Redis
 
-Appsmith runs it's own instances of MongoDB database for storing all application data, and Redis for storing session information. It's easy to switch to externally running instances of these services, by just changing the relevant environment variables.
+Appsmith runs embedded instances of MongoDB and Redis. Appsmith uses MongoDB and Redis for data storage and session management. You can use external instances by configuring the instance. External instances are stateless and help in better performance and scalability. Follow the steps on this page to set up external MongoDB and Redis for Appsmith.
```

Comment:
> Let's combine some of these lines. Hope you didn't have to change it because of the readability checker? .Also using the word "embedded" here might clash with the separate Embedded DB we are going to have for self-hosted instances which might cause confusion in the future. 
> 
> 
> 
> ```suggestion
> 
> Appsmith runs its own instances of MongoDB for storing the application data and Redis for storing session information. You can also switch to external instances, which makes the Appsmith container stateless, enabling better performance and scalability. Follow the steps on this page to set up external MongoDB and Redis for Appsmith.
> 
> ```
> 
> 

The word embedded is suggested by Sumesh. I will check the content that you suggested.